Docs/Configuration.tex

```diff
@@ -1517,44 +1516,45 @@ \subsection{Quirks Properties}\label{booterpropsquirks}
   register during their execution. This quirk requires \texttt{OC\_FIRMWARE\_RUNTIME}
   protocol implemented in \texttt{OpenRuntime.efi}.
 
-  \emph{Note}: This quirk may potentially weaken firmware security, please use
+  \emph{Note}: This quirk may potentially weaken firmware security. Please use
   \texttt{RebuildAppleMemoryMap} if the firmware supports memory attributes table (MAT).
-  Refer to \texttt{OCABC: MAT support is 1/0} log entry to determine whether MAT is supported.
+  Refer to the \texttt{OCABC: MAT support is 1/0} log entry to determine whether MAT is supported.
 
 \item
   \texttt{ForceExitBootServices}\\
   \textbf{Type}: \texttt{plist\ boolean}\\
   \textbf{Failsafe}: \texttt{false}\\
   \textbf{Description}: Retry \texttt{ExitBootServices} with new memory map on failure.
 
-  Try to ensure that \texttt{ExitBootServices} call succeeds even with outdated MemoryMap
-  key argument by obtaining current memory map and retrying \texttt{ExitBootServices} call.
+  Ensure that \texttt{ExitBootServices} call succeeds, even with an outdated MemoryMap
```

Comment:
Well, it cannot definitely ensure it, as it may still fail on second or third call. Try did make sense there.